changes/process_changes.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+
+from __future__ import absolute_import, division, print_function
+
+import argparse as A
+from funcy import memoize
```

Comment:
We generally separate imports of third-party packages from standard library’s. Can you move `funcy` to a new block below?

Reply:
OK